Firebase/Messaging/FIRMMessageCode.h

```diff
@@ -46,6 +46,7 @@ typedef NS_ENUM(NSInteger, FIRMessagingMessageCode) {
   kFIRMessagingMessageCodeAPNSTokenNotAvailableDuringTokenFetch = 2022, // I-FCM002022
   kFIRMessagingMessageCodeTokenDelegateMethodsNotImplemented = 2023, // I-FCM002023
   kFIRMessagingMessageCodeTopicFormatIsDeprecated = 2024,
+  kFIRMessagingDirectChannelConnectionFailed = 2025,
```

Comment:
This should be `FIRMessagingMessageCodeDirectChannelConnectionFailed` to match the prefix of the `enum` name.